### PR TITLE
Use crates.io libs instead of git repos

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Upgrade the dependencies to issue and verify credentials to the published libraries in crates.io.

### CODE REVIEW

File: issuer/tests/util/mod.rs [hunk 1]
```diff
@@ -12,24 +18,20 @@ use meta_issuer::groups_api::{
...
+/// Created with the Dummy Relying Party: `https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/`/
+/// Dummy alias JWS for testing, valid wrt `DUMMY_` variables
```
Comment (przydatek) [inline issuer/tests/util/mod.rs:25]: nit: why change the root key from a DUMMY value to the mainnet root?  (this way it becomes "less dummy", no?)

File: rp/frontend/static/.well-known/ii-alternative-origins [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "alternativeOrigins": [
-    RP_FE_HOSTNAME_PLACEHOLDER
+    "http://localhost:5173",
```
Comment (przydatek) [inline rp/frontend/static/.well-known/ii-alternative-origins:3]: When removing `RP_FE_HOSTNAME_PLACEHOLDER` one should update also places where it is used, namely `scripts/deploy-rp.sh` (ll. 112 and 116 in master, ll. 124 and 128 in this PR below)... 
Or is it just a leftover-change from debugging? 🤔

File: issuer/tests/issue_credentials.rs [hunk 3]
```diff
@@ -658,7 +671,15 @@ fn should_issue_vc_and_validate_e2e() -> Result<(), CallError> {
...
+        let vc_claims = claims
+            .custom()
+            .expect("missing custom claims in JWT claims")
+            .as_object()
+            .expect("malformed custom claims in JWT claims")
+            .get("vc")
+            .expect("missing vc claims in JWT custom claims")
+            .as_object()
+            .expect("malformed vc claims in JWT custom claims");
         validate_claims_match_spec(vc_claims, &spec).expect("Clam validation failed");
```
Comment (przydatek) [inline issuer/tests/issue_credentials.rs:683]: fly-by catch: typo in the error message: "Clam" -> "Claims"

File: issuer/tests/issue_credentials.rs [hunk 4]
```diff
@@ -658,7 +671,15 @@ fn should_issue_vc_and_validate_e2e() -> Result<(), CallError> {
             env.time().duration_since(UNIX_EPOCH).unwrap().as_nanos(),
         )
         .expect("credential verification failed");
-        let vc_claims = claims.vc().expect("missing VC claims");
+        let vc_claims = claims
```
Comment (przydatek) [inline issuer/tests/issue_credentials.rs:674]: nit, maybe for a future extension to `ic-verifiable-credentials`: maybe the crate should contain a helper to extract the claims?

File: rp/tests/manage_content.rs [hunk 5]
```diff
@@ -172,41 +173,32 @@ fn should_list_exclusive_content_multiple_items() {
...
...09RbHFuTGYzMzFqcmdFUlJ6VmJBS1FVVkRaTkpGNjB3ZmRoN2d3SkdjM1ZpYm1WMGd3R0RBWUlFV0NDNFM3ZHdZa3JCZEllbjVpSGRjUmJ3LUF6WXZCTWtDOVk2eVBSbVAxX3lVSU1CZ3dHQ0JGZ2daVThpS2cxM2hjbEFhMHpEQ1hEcTVLeE9tU0dScWlDWEFGb3hXVWlzNmR1REFZSUVXQ0RJQ2FMM0hFZzJlZW9JUDhvOXBZTjk0cEJ0NEhNVm5IWnJRNVktbnU3cWFJTUJnZ1JZSUpibzIzeWFLREp1NndlMkJmdTRZSVJZdGc2bUJXcU9GLU1YeGdYMGdPd3Bnd0pZSFpObHFNbnlmMEV3WUlVSU5kR0Zta0ZjWjFkYzB3d0VvWVA5Q1dNQ2d3R0RBazlqWVc1cGMzUmxjbDl5WVc1blpYT0NBMWdiMmRuM2dZSktBQUFBQUFIZ0FBQUJBVW9BQUFBQUFlX19fd0VCZ3dKS2NIVmliR2xqWDJ0bGVZSURXSVV3Z1lJd0hRWU5Ld1lCQkFHQzNId0ZBd0VDQVFZTUt3WUJCQUdDM0h3RkF3SUJBMkVBaXBHeU93bUs2MzNWNVNWeGZ3d3AwNVo4WDA1aDBMWEh4cGNYTG05V2dxcGVRajFDQjNhVnhSZXJwLU8yTnRxVUNUalgxY3J1NG13OGNjeG9pUFZDeFpadVlPcE5rZDI1V3RDWlFvTVhFSmZqSlFKcjd3QXhLTFJHUVFWc2h0QktnZ1JZSU5scEt6TzVOaUhmcGJaQlN1Y1NyUWlDb1BZcGRzTjhVT3JjcFY3MlpvbVFnZ1JZSVBOZURBbnZvcFZsZlpzekFSVF81UlYxdFN0RnNhRFJ1T3c1UmMwTlFnRTFnd0pFZEdsdFpZSURTWkRodnEzbGtPT0JHR2x6YVdkdVlYUjFjbVZZTUpqdHMtMkVKaE5NMTBfc2ZhU3hDQjg0Vm1UWWM1X0FGeWZaYmNwNUVvOUVCOHdHbmlrNW1wVXh2U0NQdnpieXoyUjBjbVZsZ3dKRGMybG5nd0pZSUd1YVFhaWIxMmFnNUxvNWdCOEZLTFlBdGhFNTY3alNWN1B6V0xFdklBc05nd0pZSUQ4WF9xUEtyaUowZ2pXdXFXZUFoZmF6aWhWWFUzcXZVVWhDZE9tNkpZTW9nZ05BIl19fQ.".to_string(),
```
Comment (przydatek) [inline rp/tests/manage_content.rs:186]: Hmm, why is this presentation much longer than the one previously used? 🤔 
I've looked inside the jws via jwt.io, and it seems that the signature-portion of the JWT-encoded VCs is much longer now -- has something changed in the signature encoding?  (I see also that now id-alias VC has an additional field `derivationOrigin`, but that doesn't cause such an increase in the size)

**Review by przydatek — APPROVED** [review]
LGTM, modulo minor issues/questions.
Thanks!

File: issuer/tests/util/mod.rs (hunk 1 above)
Comment (lmuntaner) [reply issuer/tests/util/mod.rs:25]: I agree it's less "dummy". But like this it's easy to create the jwt with the dummy relying party.

File: rp/frontend/static/.well-known/ii-alternative-origins (hunk 2 above)
Comment (lmuntaner) [reply rp/frontend/static/.well-known/ii-alternative-origins:3]: oi, this is a leftover change, thanks!!

File: issuer/tests/issue_credentials.rs (hunk 3 above)
Comment (lmuntaner) [reply issuer/tests/issue_credentials.rs:683]: Thanks!

File: rp/tests/manage_content.rs (hunk 5 above)
Comment (lmuntaner) [reply rp/tests/manage_content.rs:186]: Yes, indeed. It's the signature that makes it grow so much. This is also from mainnet. Could it be the difference between mainnet and local replica?

You can also see the same pattern in the other dummy JWS

Comment (przydatek) [reply rp/tests/manage_content.rs:186]: I see, probably the hash tree and/or delegation in the certificate/signature is larger.

**Pull request merged**